packages/web-api/src/methods.ts

```diff
@@ -188,6 +188,8 @@ import { AdminUsersSessionSetSettingsResponse } from './response/AdminUsersSessi
 import { AdminUsersSessionClearSettingsResponse } from './response/AdminUsersSessionClearSettingsResponse';
 import { AdminAppsClearResolutionResponse } from './response/AdminAppsClearResolutionResponse';
 import { AdminAppsUninstallResponse } from './response/AdminAppsUninstallResponse';
+import { OpenidConnectTokenResponse } from './response/OpenidConnectTokenResponse';
```

Comment:
If we want to adjust the name to `OpenIDConnect`, you can do it by modifying the `scripts/code_generator.rb`

Reply:
I personally prefer `OpenID` or `OpenId` but that is just a preference.